controller/test-files/event/list/ok-assignees.res.golden.json

```diff
@@ -3,21 +3,34 @@
     {
       "attributes": {
         "name": "system.assignees",
-        "newValue": "00000000-0000-0000-0000-000000000001",
-        "oldValue": "",
+        "new": null,
+        "old": null,
         "timestamp": "0001-01-01T00:00:00Z"
       },
-      "id": "00000000-0000-0000-0000-000000000002",
+      "id": "00000000-0000-0000-0000-000000000001",
       "relationships": {
         "modifier": {
           "data": {
-            "id": "00000000-0000-0000-0000-000000000001",
+            "id": "00000000-0000-0000-0000-000000000002",
             "links": {
-              "related": "http:///api/users/00000000-0000-0000-0000-000000000001"
+              "related": "http:///api/users/00000000-0000-0000-0000-000000000002"
             },
             "type": "identities"
           }
-        }
+        },
+        "newAssignees": {
```

Comment:
why call this `newAssignees` and not give it some generic name?

Reply:
Under `relationships`, we are giving more specific names. See workitems, comments, area etc.

Reply:
But how should a program consuming this know what to look for without knowing the relationships upfront?